delivery/overview.mdx

```diff
@@ -127,3 +127,36 @@ WITH (
 File sink currently supports only append-only mode, so please change the query to `append-only` and specify this explicitly after the `FORMAT ... ENCODE ...` statement.
 
 </Info>
+
+## Batching strategy for file sink
+
+The batching strategy ensures that the contents within a chunk are not further split, which enables the decoupling of the file sink. Currently, the batching strategy is available for Parquet encode. 
```

Comment:
Hi @wcy-fdu, wanna a double check if the batching strategy is available for only **parquet** or both **parquet and json**? Thanks!

Reply:
For all supported encode: json, csv and parquet.